lib/cdo/languages.rb

```diff
@@ -1,49 +1,54 @@
 require 'cdo/db'
+require 'cdo/cache_method'
 
 class Languages
-  @@table = PEGASUS_DB[:cdo_languages]
+  using CacheMethod
```

Comment:
The diff confused the method names, but these are only slightly changed- the method definitions are all prefixed by `cached `, use `table` method instead of the `@@table` class variable, and have `to_a` appended to the logic lines, otherwise they are unchanged.